themes-default/slim/views/addShows_newShow.mako

```diff
@@ -212,13 +218,33 @@ window.app = new Vue({
                 }
 
                 // Converts select boxes to command separated values [js/blackwhite.js]
-                generateBlackWhiteList(); // eslint-disable-line no-undef
+                //generateBlackWhiteList(); // eslint-disable-line no-undef
 
                 formData = new FormData(this.$refs.addShowForm);
             }
 
             this.otherShows.forEach(nextShow => formData.append('other_shows', nextShow));
 
+            // Because we're using te toggle-button.vue component, we don't have valid form input's for these.
+            // Therefor we need to add these values manually to the form data.
+
+            formData.append('subtitles', subtitles ? '1' : '0');
+            formData.append('anime', anime ? '1' : '0');
+            formData.append('scene', scene ? '1' : '0');
+            formData.append('season_folders', seasonFolder ? '1' : '0');
+
+            for (name of release.whitelist) {
+                formData.append('whitelist', name);
+            }
+
+            for (name of release.blacklist) {
+                formData.append('blacklist', name);
+            }
+
+            const statusToDesc = { 'Wanted': 3, 'Skipped': 5, 'Ignored': 7 }
```

Comment:
There's a module in the store dedicated specifically for those, `statuses`, please use it instead of hardcoded values in javascript. We should only hardcode in Python.

Reply:
No idea how to use that, sorry.

Reply:
`this.$store.state.statuses` I think?